Tests/Approval.Tests/ApiApproval.cs

```diff
@@ -19,72 +16,34 @@ namespace Approval.Tests;
 [UsesVerify]
 public class ApiApproval
 {
+    static ApiApproval() => VerifyDiffPlex.Initialize(OutputType.Compact);
+
     [Theory]
     [ClassData(typeof(TargetFrameworksTheoryData))]
     public Task ApproveApi(string frameworkVersion)
     {
-        string codeBase = Assembly.GetExecutingAssembly().Location;
-        var uri = new UriBuilder(new Uri(codeBase));
-        string assemblyPath = Uri.UnescapeDataString(uri.Path);
-        var containingDirectory = Path.GetDirectoryName(assemblyPath);
-        var configurationName = new DirectoryInfo(containingDirectory).Parent.Name;
-
-        var assemblyFile = Path.GetFullPath(
-            Path.Combine(
-                GetSourceDirectory(),
-                Path.Combine("..", "..", "Src", "FluentAssertions", "bin", configurationName, frameworkVersion,
-                    "FluentAssertions.dll")));
-
-        var assembly = Assembly.LoadFile(Path.GetFullPath(assemblyFile));
+        var configuration = typeof(ApiApproval).Assembly.GetCustomAttribute<AssemblyConfigurationAttribute>()!.Configuration;
+        var assemblyFile = GetPath("Src", "FluentAssertions", "bin", configuration, frameworkVersion, "FluentAssertions.dll");
+        var assembly = Assembly.LoadFile(assemblyFile);
         var publicApi = assembly.GeneratePublicApi(options: null);
 
         return Verifier
             .Verify(publicApi)
             .ScrubLinesContaining("FrameworkDisplayName")
             .UseDirectory(Path.Combine("ApprovedApi", "FluentAssertions"))
-            .UseStringComparer(OnlyIncludeChanges)
```

Comment:
❓ Why is this no longer needed?

Reply:
Because `VerifyDiffPlex.Initialize(OutputType.Compact);` is used instead. This is how the [Verify.DiffPlex](https://github.com/VerifyTests/Verify.DiffPlex) documentation says it should be integrated. Before this pull request, the `Verify.DiffPlex` package was referenced but not actually used at all!